manual/manual.tex

```diff
@@ -88,6 +88,13 @@
 % Read information about the repository.
 \input{env}
 
+\makeatletter
+\newcommand{\setword}[2]{%
+  \phantomsection
+  #1\def\@currentlabel{\unexpanded{#1}}\label{#2}%
+}
+\makeatother
+
```

Comment:
Perhaps add a comment what this does and why it's needed? I have no idea.

If possible, maybe move it slightly higher, above the `\input{env}` with the other `\newcommand`.

Reply:
This block of instructions provides hyperrefs within the pdf. E.g. An error might say ```See seL4_IllegalOperation in common error codes.``` **seL4_IllegalOperation** links to **10.1.11 Common Error Codes**. Without the enveloping ```\makeatletter``` and ```\makeatother``` the line would read ```See 10.1.11 in common error codes.```

Reply:
So similar to `\apifunc`, but more generic.